src/Components/HeaderContent.js

```diff
@@ -36,27 +45,12 @@ const Header = () => {
       >
         Benvenuto su Jewelry Shop Olivia Rossi!
       </p>
-      <Router>
-        <Routes>
-          <Route
-            exact
-            path='/login'
-            element={<Login afterLogin={setUserNameAfterLogin} />}
-          />
-          <Route exact path='/register' element={<Register />} />
-          <Route exact path='/logout' element={<Logout />} />
-          <Route
-            path='*'
-            element={
-              userName ? <p className='accountPar'>Welcome {userName}!</p> : ''
-            }
-          ></Route>
-        </Routes>
-      </Router>
 
       <a href='##' className='carello-button'>
         <img className='carelloImg' src='Carello.png' alt='logo' />
       </a>
+
+      {username ? <WelcomeMessage username={username} /> : <NotLoggedInButtons />}
```

Comment:
This is a [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator), it is used really commonly in React code since it works as a conditional and returns the result of the conditional check. Regular `if/else` functions don't work inside JSX (ie: inside the `return` value of a React function component) because `if/else` by itself doesn't _return_ anything unless they're inside a function. 

If you prefer if/else statements, then you have to extract it into a function: https://react.dev/learn/conditional-rendering

So we could have:

```jsx
const Header = ({ username }) => {
  let text = useRef(null)

  useEffect(() => {
    gsap.to(text, {
      duration: 8,
      delay: 3,
      opacity: 1,
      repeat: -1
    })
  }, [])

  function generateHeaderContentRightSide() {
    if (username) {
      return <WelcomeMessage username={username} />
    } else {
      return <NotLoggedInButtons />
    }
  }

  return (
    <div className='headerContainer'>
      <img className='logoImg' src='Logo.png' alt='logo' />

      <p
        ref={(el) => {
          text = el
        }}
        className='par'
      >
        Benvenuto su Jewelry Shop Olivia Rossi!
      </p>

      <a href='##' className='carello-button'>
        <img className='carelloImg' src='Carello.png' alt='logo' />
      </a>

      { generateHeaderContentRightSide() }
    </div>
  )
}
```